### DISCUSSION TITLE
사다리와 벽 같은 물체를 올라갈수 있는 기능이 있을까요?

### DISCUSSION THREAD

**Discussion by sul321** [naverz/zepeto-studio-kor#1216] [5. Q&A]
안녕하세요
개발은 처음 이여서 가이드만 보고 따라하다가 없는 부분이 있어 질문 드립니다 
사다리 메쉬를 이용해서 위로 올라가는 것을 만들고 싶은데 가능한가요?
또 처음 캐릭터 스폰위치가 0,0,0에서 다른 위치로 변경을 시켜도 계속 플레이 하면 그 자리로 돌아와서요 이 부분도 알려주실수 있을까요?

**Comment by ckdlskgkdrhd** [comment]
you might wanna check this link down below.
https://github.com/naverz/zepeto-studio-global/discussions/52

**Reply by ckdlskgkdrhd** [reply]
or you could adjust this 
![adjust this](https://user-images.githubusercontent.com/110361648/186324981-b770a626-4fc4-459c-b406-938fe64ededa.PNG)
